channels/serializers/channels.py

```diff
@@ -194,4 +195,9 @@ def update(self, instance, validated_data):
             channel_obj.banner.save(f"channel_banner_{name}.jpg", banner, save=False)
             channel_obj.save(update_fields=["banner"], update_image=True)
 
+        about = validated_data.get("about")
+        if about:
```

Comment:
Is there ever going to be a need to null out the about value? Or would you just end up setting it to `[]`?

Reply:
hmm I suppose we might want users to be able to null it out, in case a channel wanted to no longer have an about page.

Reply:
ok, I rewrote this logic to check if there is a value for `about` in the validated data and, if so, save that as the new value for about. I figure if we're going to delete the value of the field we shouldn't just look for its absence from the request, but require the user to make a request with `about: null` in it.